packages/gatsby-admin/README.md

```diff
@@ -2,11 +2,50 @@
 
 A visual interface to configure your Gatsby site.
 
-## Development
+## Architecture
+
+The Gatsby Admin interface (this package) is a standard Gatsby site.
+
+It uses [theme-ui](https://theme-ui.com) (with the [strict-ui](https://github.com/system-ui/theme-ui/pull/719) experimantal extension) and [gatsby-interface](https://github.com/gatsby-inc/gatsby-interface) for styling.
+
+It fetches its data from the [gatsby-recipes GraphQL server](../gatsby-recipes/src/graphql-server/), which exposes all the information we need about the locally running Gatsby site, using [urql](https://github.com/FormidableLabs/urql).
```

Comment:
You'll need to change all the relative links (e.g. `../gatsby-recipes/src/graphql-server/`) to absolute GitHub links as the README can/will also be read on our website or NPM or wherever :)

Reply:
Great call, thank you @LekoArts!